shared/containers/Community/index.js

```diff
@@ -1,6 +1,14 @@
 import { connect } from 'react-redux';
 import Community from '../../components/Community';
 
+function featuredEvent(events) {
+  return events.find(e => e.displayLevel === 'Featured') || {};
+}
+
+function futureEvents(events) {
+  return events.filter(e => e.displayLevel === 'Highlighted').slice(0, 3);
```

Comment:
I'll add a `MAX_EVENTS` constant since it's just truncating to an array of 3 rather than trying to get a specific slice.